src/store/actions.ts

```diff
@@ -1132,14 +1136,205 @@ export const fetchMRASProfile: ActionIF = async ({ commit, getters }): Promise<a
   return null
 }
 
+/** Submits an edited NR or a new name submission. */
+export const submit: any = async ({ commit, getters }): Promise<any> => {
```

Comment:
Need to refactor this (after merge). I don't think a store action should **ever** make a network call !!!

If this functionality is truly needed, there should be a high-level function that does the store update and the network call. Maybe in a mixin?